mypy_protobuf/main.py

```diff
@@ -270,25 +270,23 @@ def _write_comments(self, scl: SourceCodeLocation) -> bool:
             lines.extend(self._break_text(sci_loc.trailing_comments))
 
         lines = [
-            # Replace backslashes with double-backslashes
-            # to remove anything that would become an escape sequence in a Python string.
             # Escape triple-quotes that would otherwise end the docstring early.
-            line.replace("\\", "\\\\").replace('"""', '\\"\\"\\"')
+            line.replace('"""', '\\"\\"\\"')
```

Comment:
Use a raw string declaration so you do not need to escape `\`
```suggestion
            line.replace('"""', r'\"\"\"')
```